Makefile

```diff
@@ -36,6 +36,10 @@ test-unit:
 test-e2e:
 	./test/e2e-tests.sh
 
+# TODO: that will - soon... run the Kafka operator e2e hook
+test-e2e-all:
```

Comment:
What's the supposed to be difference of this vs. `test-all-e2e` further below in this file?